### ISSUE TITLE
Getting "missing references errors" in Visual Code and Omnisharp using dotnet script.

### ISSUE THREAD

**Issue by mmarvb8h** [OmniSharp/omnisharp-vscode#5691]
Please paste the output from your clipboard

## Expected Behavior ##

Should not be getting 'unknown references' errors from editor.

## Actual Behavior ##

Creating application using dotnet script with Visual Code.

In the editor window i see the below errors with red underlines.

The type or namespace name 'AspNetCore' does not exist in the namespace 'Microsoft' (are you missing an assembly reference?)

The name 'WebApplication' does not exist in the current context

script does contain the nuget references:

#r "nuget: Microsoft.AspNetCore.App, 7.0.5"

The application runs fine. So i'm assuming the issue is in Intellisense. Have i set up something wrong?

### OmniSharp log ###
<details>Post the output from Output-->OmniSharp log here</details>

### C# log ###
<details>Post the output from Output-->C# here</details>

## Environment information ##

**VSCode version**: 1.78.2
**C# Extension**: 1.25.7

<details><summary>Mono Information</summary>

thx ~Marvin Foster  
marvinfoster@sage.com. , mmarvb7@gmail.com
 
</details>
<details><summary>Dotnet Information</summary>
.NET SDK:
 Version:   7.0.100
 Commit:    e12b7af219

Runtime Environment:
 OS Name:     Mac OS X
 OS Version:  12.6
 OS Platform: Darwin
 RID:         osx.12-arm64
 Base Path:   /usr/local/share/dotnet/sdk/7.0.100/

Host:
  Version:      7.0.0
  Architecture: arm64
  Commit:       d099f075e4

.NET SDKs installed:
  6.0.402 [/usr/local/share/dotnet/sdk]
  7.0.100 [/usr/local/share/dotnet/sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 6.0.10 [/usr/local/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 7.0.0 [/usr/local/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 6.0.10 [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]
  Microsoft.NETCore.App 7.0.0 [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]

Other architectures found:
  None

Environment variables:
  Not set

global.json file:
  Not found

Learn more:
  https://aka.ms/dotnet/info

Download .NET:
  https://aka.ms/dotnet/download
</details>
<details><summary>Visual Studio Code Extensions</summary>

|Extension|Author|Version|
|---|---|---|
|csharp|ms-dotnettools|1.25.7|;

</details>

**Comment by JoeRobich** [comment]
CSX support is provided by OmniSharp-Roslyn. According to the dotnet-script readme. OmniSharp needs to be restarted in order to pick up the new package reference. From the VS Code command bar you can run "OmniSharp: Restart OmniSharp".
<img width="584" alt="image" src="https://github.com/OmniSharp/omnisharp-vscode/assets/611219/2ccccbdd-9a6c-4bfc-a5f0-35fe931defe6">

**Comment by mmarvb8h** [comment]
Tried ur suggestion. Errors disappear for about 30 to 60 seconds then return.

**Comment by JoeRobich** [comment]
I don't see that version of the package on nuget.org. https://www.nuget.org/packages/Microsoft.AspNetCore.App#versions-body-tab

**Comment by JoeRobich** [comment]
Maybe you are looking to specify the Web SDK? https://github.com/dotnet-script/dotnet-script#specifying-an-sdk

**Comment by mmarvb8h** [comment]
Does not work as they described in the readme. I have an open issue with dotnet-script. (https://github.com/dotnet-script/dotnet-script/issues/723)

I then created a minimal Webapi and used that as a starting point. All seems to work except for Intellisense which sometimes seems to have problems. 

<Solved> While responding i noticed that the .csprog file used by dotnet-script is missing the tag <ImplicitUsings>enable<ImplicitUsings>. I removed from my Webapi app and received the same Intellisense wornings during build. If i choose to ignore, app runs anyway. So that appears to be the issue. 

So i will close issue.

thx for looking at this...
